ena-submission/scripts/test_ena_submission.py

```diff
@@ -115,54 +131,52 @@ class ProjectCreationTests(unittest.TestCase):
     def test_create_project_success(self, mock_post) -> None:
         # Testing successful project creation
         mock_post.return_value = mock_requests_post(200, test_project_xml_response)
-        project_set = default_project_type()
-        response = create_ena_project(test_ena_config, project_set)
+        project_set = default_project_set()
+        response = create_ena_project(MOCK_CONFIG, project_set)
         desired_response = {
             "bioproject_accession": "PRJEB20767",
             "ena_submission_accession": "ERA912529",
         }
-        self.assertEqual(response.result, desired_response)
+        assert response.result == desired_response
 
     @mock.patch("requests.post")
     def test_create_project_xml_failure(self, mock_post):
         # Testing project creation failure due to incorrect status
         mock_post.return_value = mock_requests_post(200, test_project_xml_failure_response)
-        project_set = default_project_type()
-        response = create_ena_project(test_ena_config, project_set)
+        project_set = default_project_set()
+        with self.assertLogs("ena_deposition.ena_submission_helper", level="WARNING") as cm:
+            response = create_ena_project(MOCK_CONFIG, project_set)
+            self.assertIn("Response is in unexpected format", cm.output[0])
         error_message_part = "Response is in unexpected format"
-        self.assertIn(error_message_part, response.errors[0])
+        assert error_message_part in response.errors[0]
 
     @mock.patch("requests.post")
     def test_create_project_server_failure(self, mock_post):
         # Testing project creation failure
         mock_post.return_value = mock_requests_post(500, "Internal Server Error")
-        project_set = default_project_type()
-        response = create_ena_project(test_ena_config, project_set)
+        project_set = default_project_set()
+        with self.assertLogs("ena_deposition.ena_submission_helper", level="WARNING") as cm:
+            response = create_ena_project(MOCK_CONFIG, project_set)
+            self.assertIn("Request failed with status:500", cm.output[0])
         error_message_part = "Request failed with status:500"
-        self.assertIn(error_message_part, response.errors[0])
+        assert error_message_part in response.errors[0]
```

Comment:
why is this change required?

Reply:
not required but now we're switching to pytest it uses normal asserts 

https://docs.pytest.org/en/stable/how-to/assert.html

https://docs.pytest.org/en/stable/how-to/unittest.html